runtime/client/src/clientmgr.cpp

```diff
@@ -281,7 +281,7 @@ LTRESULT CClientMgr::Init(const char **resTrees, uint32 nResTrees, uint32 nNumCo
 
     // Setup the ClientShellDE.
     dResult = dsi_InitClientShellDE();
-    if (dResult != LTTRUE)
+    if (dResult != LT_OK)
```

Comment:
Will need to fix the logic in sys/linux/linuxdsys.cpp:LTRESULT dsi_InitClientShellDE() to follow the logic